src/playground/src/utils.ts

```diff
@@ -19,14 +19,9 @@ export function handleShareLink(onContents: (contents : string | null) => void):
 }
 
 export function copyShareLinkToClipboard(content: string): void {
-  document.addEventListener('copy', function onCopy(e: ClipboardEvent) {
-    const contentHash = encodeHash(content);
-    e.clipboardData.setData('text/plain', `https://aka.ms/bicepdemo#${contentHash}`);
-    e.preventDefault();
-    document.removeEventListener('copy', onCopy, true);
-  });
-
-  document.execCommand('copy');
+  const contentHash = encodeHash(content);
+  const shareLink = `https://aka.ms/bicepdemo#${contentHash}`;
+  navigator.clipboard.writeText(shareLink);
```

Comment:
Uses the proper [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API) instead of the old way with events.

The issue with events was probably that the event was never properly removed, so it kept pushing the link in the clipboard repeatedly.